content/blog/2024-06-20-why-rse.md

```diff
@@ -0,0 +1,100 @@
++++
+title = "Small steps towards sustainable and open research"
+
+[extra]
+
+author = "Aurélie Vancraeyenest, Samantha Wittke"
++++
+
+# Small steps towards sustainable and open research
+- or why you should care about Research Software Engineering
+
+A blog post summarizing the outcome of a discussion at Nordic-RSE conference May 2024 (https://nordic-rse.org/events/2024-in-person-conference/).
+
+#### What is a Research Software Engineer (RSE)?
+
+**The short answer** 
+People striving for "Better software, better research" 
+
+**The long answer**
+*Definition of Research Software ? (from Martin's slides)*
+Research software is software that one use to generate, process or analyse results that is intended to appear in a publication. It can be anything from a few lines of code, to a professionally developed software package.
+
+A Research Software Engineer is a person who combines an intricate understanding of research with professional software engineering expertise. RSE often have background in academia and have specialized in computing related areas, be it either software development, data analysis, (high-performance) computing, etc.
+
+Independently of their area of expertise, their common goal is better maintainability, sustainability of research software and better reproducibility of research outputs.
+
+**What an RSE is not**
+
+Even though they are in close collaboration with IT services, RSEs are not IT support people, as in IT manager or IT system specialists. They have a dual background (research/coding), they take part in research projects and speak the "same language" as researchers. 
+
+They also are not enforcers, they don't enforce technologies or methods but build on the tools already in use in research groups.  
+
+#### Why would an organization need RSEs? - Current state of Research software 
+
+Does any of following scenario sound familiar? 
+
+* A PhD/post doc has developed some really nice piece of code, but they left and no one knows anymore how to run it or develop it further. 
+* Someone wrote some code long time ago that is used daily by a research group, but no one knows the technology in the research group anymore and it cannot be modified, updated or compiled anymore. 
+* When a newcomer joins a team and starts familiarizing with analysis/modelling/etc tools, the only way is to "ask others who use it". No document or tutorials are available. 
+* Some essential piece of software runs on a very old machine or on very old version of language and it's very difficult to replace or update, if even possible. This can even lead as far as stopping research activities in the team for some time 
+* After an article is submitted, one reviewer ask for additional analysis, or slightly different plots, but upon running the code again, one cannot reproduce the plots/results anymore. 
+
+#### How RSE can help ? 
+
+**Short answer**
+
+Improve FAIRness of research software and by that save time and resources. 
```

Comment:
not quite sure if this is the angle we should take. Having all on board on "FAIR" is wishful thinking and I think RSE is actually broader than FAIR (talking basically of Open Science here). It is perfectly feasible for RSE to help on internal or closed code which will never be published/distributed. 

In my understanding this should become a piece which you can hand to any Prof/Leader, so (s)he should also be on board if they are not sigining up on the FAIR thing.

So perhaps: Improve the (re)usability of research software - including improving performance, reduce time and resource usage and ease on-boarding of new users.

Reply:
```suggestion
Improve the (re)usability of research software - including improving performance, reduce time and resource usage and ease on-boarding of new users.
```
Agree with this suggestion